app/Http/Controllers/SignLocallyController.php

```diff
@@ -198,6 +198,7 @@ public function startSignCustomFile(Request $request)
             Session::put("issandbox-$fileId", true);
             return redirect()->to(url('/show-download-signed-file') . "?file_id=$fileId");
         } else {
+            Cache::put("prepared-files-$docId", $sourceFilesCache ?? null);
```

Comment:
sourceFilesCache is undefined here and breaking the app

Reply:
with nullish coalescin the sourceFilesCache can not be undefined

Reply:
I updated it and set $sourceFilesCache to null for better readability.